deployments/traefik.go

```diff
@@ -136,15 +138,16 @@ func (k Traefik) GetVersion() string {
 
 func (k Traefik) Deploy(c *kubernetes.Cluster, ui *termui.UI, options kubernetes.InstallationOptions) error {
 
-	skipTraefik, err := options.GetBool("skip_traefik", TraefikDeploymentID)
+	skipTraefik, err := options.GetBool("skip-traefik", TraefikDeploymentID)
```

Comment:
not sure if this is the right place to replace all `_` in flags with `-`. We can do it in another PR. I think it's more common to use `-` in flag names.

Reply:
okie